### DISCUSSION TITLE
Debunked: Some of the misconceptions cosmologists have

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#48] [Metaphysics and Philosophy]
***Was the universe made for us?*** | Sabine Hossenfelder on [YouTube](https://youtu.be/V45ABEM-7m0)

Some of the nonsensical questions discussed by cosmologists.

- Was the universe made for us? ***No***
- Could complex structures (necessary for life) exist if the fundamental constants were different? ***Yes***
- Are *Fine tuning arguments* meaningful? ***No***, the probability distribution of fundamental constants is not known.
- Is *complexity* a rare phenomenon? ***No***, complexity is more likely the rule.
- Do copies of us exist (in the multiverse)? ***No***, it's an *unscientific* question.
- In Big Bang cosmology, is the universe finite?  ***No*** the universe is always infinitely large in Big Bang cosmology.
- Can copies of us exist in the *Many Worlds interpretation* of quantum mechanics? ***No***, it doesn't make sense to speak of such copies.
- If some mathematical equations are correct, does it mean they describe something real? ***No***
- Can particles think (in the panpsychism sense)? ***No***
- Do physicists who consider how the universe began use extensions of religious and spiritual ideas? ***Yes***
- Can science answer every questions? ***No***, the way science works some things may remain forever unknown.

**Comment by Pierre314159** [comment]
Excellent talk.

However, I got disappointed when Sabine had to resort to the expansion of the universe (calling THAT « science ») as a counter argument to the idea that there could be copies of ourselves in quantum « Many Worlds », a multiverse, or whatever other synonym you can have of an infinite and eternal universe… Isn’t it sufficient to say that infinities not being measurable you step out of science the instant one is involved?

Anyways, the confabulation about « Copies of ourselves » self-debunks to anyone who understands infinity the way Cantor defined it: an infinite number of people is a first-order infinity, while the number of POSSIBLE people is a third-order infinity. Meaning that for each individual person, there is an infinity of other POSSIBLE individual persons, meaning that the probability of having one other identical person is 1 divided by infinity: zero. A « copy of ourselves » is a mathematical impossibility. « Infinite » is not synonym of « everything », and this is the source of the confusion for many people…

Then comes the contradiction: « Correct mathematical expressions do NOT describe something real ». Therefore, the « correct mathematical expression » above does NOT mean that there exist no copies of ourselves!!!

**Reply by RedshiftDrift** [reply]
Indeed Hossenfelder has been criticized for labeling some activities as *non scientific* and then go back to her office to work on quantum loop gravity...

It is correct that some of her arguments rely on the Big Bang model, but it doesn't have to be so (in some cases at least!)

**Reply by poelzi** [reply]
This is indeed the question if reality uses mathematical or classical logic. A question that we used to discuss in the 60s, before QM started to dominate the field.
I'm very happy that BSM-SG only requires classical logic, which easily stops most of the theoretical none-sense like multiverses, worm holes or tachions.
